pages/gallery.html

```diff
@@ -4,9 +4,59 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="../styles/gallery.css" rel="stylesheet">
+  <link href="../styles/style.css" rel="stylesheet">
+  <title>Pet Gallery</title>
 </head>
 <body>
-  
+  <header>
+    <nav>
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="facts.html">Fun Facts</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+    <h1>Pet Gallery</h1>
+    <div class="flex-container">
+      <div class="flex-item">
+        <figure>
+          <img src="../images/puppies-kitten.jpg" alt="Puppies &amp; Kittens Image">
+          <figcaption>
+          <h2>Puppies &amp; Kittens</h2>
+          <p>Puppies and kittens are the top two most popular pets in the world.</p>
+          </figcaption>
+        </figure>
+      </div>
+      <div class="flex-item">
+        <figure>
+          <img src="../images/hamsters.jpg" alt="Hamsters Image">
+          <figcaption>
+          <h2>Hamsters</h2>
+          <p>Hamsters are interesting pets with unique behavior, like being active during dawn and dusk.</p>
+          </figcaption>
+        </figure>
+      </div>
+      <div class="flex-item">
```

Comment:
You use a lot of divs in this html. In the future see if you can rely less on the divs and more on other semantic elements like section or article.